documentation/airport-format.md

```diff
@@ -1,179 +1,168 @@
----
-title: Airport format
----
-[back to index](index.html)
+# Airport Format
 
-# Airport format
+[NOTICE](#NOTICE)
+[Template](#Template)
+[Comments](#Comments)
+
+## DEVNOTES
+
+feature/184 - version 0.2 documentation
+Working on formatting this document like the [spawnPatterns readme.](openscope/src/assets/scripts/client/trafficGenerator/spawnPatternReadme.md).
+Plans:
+    - Remove all "// <explanation>" lines from the template so it can be copy/pasted.
+    - Explain needed properties below the template.
+
+## NOTICE
 
 The airport JSON file must be in `assets/airports`; the filename
 should be `icao.json` where `icao` is the lowercase four-letter ICAO
 airport code, such as `ksfo` or `kmsp`.  If this is a new airport, there
-should also be an entry added to the end of [`airportLoadList.js`](../src.assets/scripts/airport/airportLoadList.js)
+should also be an entry added to the end of [`airportLoadList.js`](../src.assets/scripts/airport/airportLoadList.js).
 See the comments for information on the correct structure to use.
+
+## Template
 ```
 {
   "radio": {
-    "twr": "controller callsign for tower",
-    "app": "controller callsign for approach control",
-    "dep": "controller callsign for departure control"
+    "twr": "tower callsign",                // The callsign of the airport control tower.
+    "app": "approach control callsign",     // The callsign of the airport approach tower.
```

Comment:
Let's do:
 - `callsign of the facility providing tower services`
 - `callsign of the facility providing approach control services`
 - `callsign of the facility providing departure control services`